themes/classic/_dev/css/partials/_variables.scss

```diff
@@ -3,6 +3,7 @@ $gray-darker: #232323;
 $gray: #7A7A7A;
 $gray-light: #F1F1F1;
 $gray-light-second: #DBDBDB;
+$gray-light-third: #eee;
```

Comment:
I'm unsure we need to a a variable for this task.

Reply:
No problem,I will take it off.